content/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry.md

```diff
@@ -199,7 +199,7 @@ Using packages from {% data variables.product.prodname_dotcom %} in your project
 
 {% data reusables.package_registry.authenticate-step %}
 
-2. To use a package, add `ItemGroup` and configure the `PackageReference` field in the *.csproj* project file, replacing the `OctokittenApp` package with your package dependency and `1.0.0` with the version you want to use:
+2. To use a package, add `ItemGroup` and configure the `PackageReference` field in the *.csproj* project file, replacing the `OctokittenApp` package with your package dependency and `12.0.2` with the version you want to use:
```

Comment:
Adding to your proposed change, let's mention the two attributes to be changed by their name and value, to try and avoid confusion about what needs to be changed in this step.

```suggestion
2. To use a package, add `ItemGroup` and configure the `PackageReference` field in the *.csproj* project file. Replace the `OctokittenApp` value in `Include="OctokittenApp"` with your package dependency, and replace the `12.0.2` value in `Version="12.0.2"` with the version you want to use:
```